engineering-report/document.adoc

```diff
@@ -0,0 +1,62 @@
+= Engineering report for OGC (add title text)
+:comment: ### Document type; mandatory. Visit: https://www.metanorma.com/author/ogc/authoring/ for permitted types
+:doctype: engineering-report
+:comment: ### Language of the document; mandatory. Specified in two-letter code: "en" for English, "fr" for French
+:language: en
```

Comment:
No evidence that OGC publishes outside of English. "en" is supplied by default; I would leave the attribute out, unless we know for a fact that OGC does publish outside of English.

Reply:
Ok, done.